src/mono/wasm/templates/templates/console/runtimeconfig.template.json

```diff
@@ -4,12 +4,10 @@
             {
                 "name": "node",
                 "js-path": "main.mjs",
-                "Host": "nodejs"
-            },
-            {
-                "name": "v8",
-                "js-path": "main.mjs",
-                "Host": "v8"
+                "Host": "nodejs",
+                "host-args": [
+                    "--experimental-wasm-simd"
+                ]
```

Comment:
why does this diff show v8 being removed?

Reply:
@radical is that intentional?

Reply:
We don't support v8 for the console template, only nodejs.

Reply:
so this is unrelated to SIMD, right?